### DISCUSSION TITLE
Group Assignment teams cannot be created

### DISCUSSION THREAD

**Discussion by lizajane999** [community/community#67910] [GitHub Education] [answered]
### Hi everyone,

No teams can be created, even by myself (the owner of the Classroom). The "Create team" button yields this error message:

![Screen Shot 2023-09-21 at 12 46 16 PM](https://github.com/community/community/assets/279775/34dea83e-6e3a-4f78-80be-ba897cfb80f5)

I have been teaching using GitHub Classroom for years and have never had this happen before. I just tried again after over an hour and it is still broken. I had to rapidly make a new individual assignment and add students in a group to an individual repository.

## SOLVED!
I discovered that teams from previous semesters had built up to a large number. I don't know if it was just that there were too many teams or that the names we tried were all already used, but when I deleted all previous semesters' teams I was able to make new ones again.

**Comment by xihuixu** [comment]
I am having the same issue. Can someone look at this bug? It is almost final season.

**Comment by lizajane999** [comment, marked as answer]
See original post. Delete all your old teams and it should fix this.

**Comment by ayundarevou** [comment]
Hello @lizajane999 may I know how long did it take until you are able to create team?
Because I have deleted all the teams from the previous batch for over an hour ago, however the exact problem still occurred
Not sure what happened here
<img width="1420" alt="Screenshot 2024-04-18 at 21 29 45" src="https://github.com/community/community/assets/124788015/1cb08ab7-598c-4ec7-a29f-76f535c4dee2">

**Comment by jennirinker** [comment]
For anyone else who wanders across this thread: I think the issue is due to same team names across assignments. I made a thread here for that discussion: https://github.com/orgs/community/discussions/155223
